presto-verifier/src/main/java/com/facebook/presto/verifier/VerifyCommand.java

```diff
@@ -0,0 +1,412 @@
+
+        ImmutableList.Builder<Module> builder = ImmutableList.<Module>builder()
+                .add(new PrestoVerifierModule())
+                .addAll(getAdditionalModules());
+
+        Bootstrap app = new Bootstrap(builder.build());
+        Injector injector;
+        try {
+            injector = app.strictConfig().initialize();
+        }
+        catch (Exception e) {
+            throwIfUnchecked(e);
+            throw new RuntimeException(e);
+        }
+
+        try {
+            VerifierConfig config = injector.getInstance(VerifierConfig.class);
+            injector.injectMembers(this);
+            Set<String> supportedEventClients = injector.getInstance(Key.get(new TypeLiteral<Set<String>>() {}, Names.named(SUPPORTED_EVENT_CLIENTS)));
+            for (String clientType : config.getEventClients()) {
+                checkArgument(supportedEventClients.contains(clientType), "Unsupported event client: %s", clientType);
+            }
+            Set<EventClient> eventClients = injector.getInstance(Key.get(new TypeLiteral<Set<EventClient>>() {}));
+
+            VerifierDao dao = getQueryDatabase(injector).onDemand(VerifierDao.class);
+
+            ImmutableList.Builder<QueryPair> queriesBuilder = ImmutableList.builder();
+            for (String suite : config.getSuites()) {
+                queriesBuilder.addAll(dao.getQueriesBySuite(suite, config.getMaxQueries()));
+            }
+
+            List<QueryPair> queries = queriesBuilder.build();
+            queries = applyOverrides(config, queries);
+            queries = filterQueryTypes(new SqlParser(getParserOptions()), config, queries);
+            queries = filterQueries(queries);
+            if (config.getShadowWrites()) {
+                Sets.SetView<QueryType> allowedTypes = Sets.union(config.getTestQueryTypes(), config.getControlQueryTypes());
+                checkArgument(!Sets.intersection(allowedTypes, ImmutableSet.of(CREATE, MODIFY)).isEmpty(), "CREATE or MODIFY queries must be allowed in test or control to use write shadowing");
+                queries = rewriteQueries(new SqlParser(getParserOptions()), config, queries);
+            }
+
+            // Load jdbc drivers if needed
+            if (config.getAdditionalJdbcDriverPath() != null) {
+                List<URL> urlList = getUrls(config.getAdditionalJdbcDriverPath());
+                URL[] urls = new URL[urlList.size()];
+                urlList.toArray(urls);
+                if (config.getTestJdbcDriverName() != null) {
+                    loadJdbcDriver(urls, config.getTestJdbcDriverName());
+                }
+                if (config.getControlJdbcDriverName() != null) {
+                    loadJdbcDriver(urls, config.getControlJdbcDriverName());
+                }
+            }
+
+            // TODO: construct this with Guice
+            int numFailedQueries = new Verifier(System.out, config, eventClients).run(queries);
+            System.exit((numFailedQueries > 0) ? 1 : 0);
```

Comment:
Basically I inline the original behavior instead of return the `numFailedQueries`. Would like to see if there are any better options.